README.md

```diff
@@ -22,35 +23,61 @@ A simple, lightweight JavaScript API for handling cookies
 
 ## Installation
 
+### NPM
+
+JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) under the name `js-cookie`.
+
+```
+$ npm install js-cookie --save
+```
+
 ### Direct download
 
-Download the script [here](https://github.com/js-cookie/js-cookie/blob/latest/src/js.cookie.js) and include it (unless you are packaging scripts somehow else):
+The source comes as an ES module. If you download it [here](https://github.com/js-cookie/js-cookie/blob/latest/src/js.cookie.mjs) directly, you must include it as such.
+
+Example:
 
 ```html
-<script src="/path/to/js.cookie.js"></script>
+<script type="module" src="./js.cookie.mjs"></script>
+<script type="module">
+  import Cookies from "./js.cookie.mjs";
+
+  Cookies.set('foo', 'bar');
+</script>
 ```
 
-Or include it via [jsDelivr CDN](https://www.jsdelivr.com/package/npm/js-cookie):
+*Not all browsers support ES modules natively yet*. For this reason the npm package/release
+comes with both an ES module as well as an UMD module variant. Include the module along
+with the fallback to account for this:
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
+<script type="module" src="/path/to/js.cookie.mjs"></script>
```

Comment:
Hmmm enforcing consumers to use a JS file with a `.mjs` file extension at this level seems a little odd. Would all browsers know how to interpret a .mjs file extension?

Reply:
I didn't mean to force consumers, more like provide an example of how to include the module of that package directly, along with the fallback. In general, browsers do not care about the extension, though it's important a server serves `mjs` files with the proper mime type of `text/javascript`.

Generally I'm assuming the more likely use case for ES modules is to be bundled up, so I might add that here...

Reply:
Or do you mean it's too early to use `mjs` files in general? I was looking at https://v8.dev/features/modules and it seemed the way to go..

Reply:
It seems the value for outputting a `.mjs` extension is to assist on documentation. Should we stick with one file for simplicity? Is there another reason for having both `.js` and `.mjs`?

Reply:
For now I'm siding with the node developers and wish to keep the extension:

> Still, we recommend using the .mjs extension for modules, for two reasons:
>
> 1. During development, the .mjs extension makes it crystal clear to you and anyone else looking at your project that the file is a module as opposed to a classic script. (It’s not always possible to tell just by looking at the code.) As mentioned before, modules are treated differently than classic scripts, so the difference is hugely important!
> 2. It ensures that your file is parsed as a module by runtimes such as Node.js and d8, and build tools such as Babel. While these environments and tools each have proprietary ways via configuration to interpret files with other extensions as modules, the .mjs extension is the cross-compatible way to ensure that files are treated as modules.

I'm assuming it will be rather rare that the source will be consumed directly, rather than being bundled up with a tool. In the latter case the extension matters less, stressing the aforementioned arguments further.

Let's see how it goes. We can always change that back later on, but let's get some feedback first and start releasing a v3 beta once we're ready...

Reply:
Sounds good, if it's a convention it doesn't hurt simply following it.